ui/imports/shared/controls/chat/ChatInputLinksPreviewArea.qml

```diff
@@ -8,22 +8,29 @@ import StatusQ.Core 0.1
 import shared.status 1.0
 import shared.controls.chat 1.0
 
+import utils 1.0
+
 import SortFilterProxyModel 0.2
 
 Control {
     id: root
 
     required property var imagePreviewModel
```

Comment:
Not part of this pr, but what is expected here is not a model but js array, what is quite misleading from component's api perspective. It would be nice to rename it or add appropriate comment.